src/client/bridge-data.ts

```diff
@@ -1,7 +1,10 @@
 import { EthAddress } from "@aztec/barretenberg/address";
 
+// @dev Parameter assetId is an index of the corresponding address in the array returned from
+//      RollupProcessor.getSupportedAssets() incremented by 1 (the increment is there because ETH has id 0 and it is
+//      not present in the array)
 export interface AssetValue {
-  assetId: bigint; // the Aztec AssetId (this can be queried from the rollup contract)
+  assetId: bigint;
```

Comment:
Instead of giving description of the array etc, might just note that the address is retrieved from `RollupProcessor.getSupportedAsset(assetId)`.